applications/ShapeOptimizationApplication/test_examples/01_Strain_Energy_Minimization_3D_Hook/run_optimization.py

```diff
@@ -34,13 +29,15 @@
 ###TODO replace this "model" for real one once available in kratos core
 Model = {ProjectParameters["problem_data"]["model_part_name"].GetString() : main_model_part}
 
-# Create an optimizer  
+# Create an optimizer 
 # Note that internally variables related to the optimizer are added to the model part
-optimizer = optimizerFactory.CreateOptimizer( main_model_part, optimizationSettings )
+optimizerFactory = __import__("optimizer_factory")
```

Comment:
This way of importing modules looks a little strange to me, is there a reason not to use the standard
`import optimizer_factory as optimizerFactory ` ?